department-of-veterans-affairs/va.gov-cms
# VACMS-7076 set up design tokens

## Description

Relates to #7076 + #7226 

Sets up some initial variables tied to the cms design system tokens, overrides existing claro variables with them. This gives us a lot of flexibility to leverage claro in a sustainable maintainable way.

## Testing done

- Checked log in, knowledge base, user profile page, various content type edit forms, section views, content view. Colors look to have cascaded down properly in most/all cases. If edge cases appear, or design asks for changes, we can easily target those in future PRs.

## Screenshots
<img width="1916" alt="Screen Shot 2021-12-10 at 4 27 08 PM" src="https://user-images.githubusercontent.com/11279744/145657070-5cc3287b-21dd-4de2-a609-a97bd19dd81a.png">
<img width="1917" alt="Screen Shot 2021-12-10 at 4 27 35 PM" src="https://user-images.githubusercontent.com/11279744/145657072-ea87e27f-b2f5-427d-a41b-33f26645b46b.png">
<img width="1188" alt="Screen Shot 2021-12-10 at 4 30 13 PM" src="https://user-images.githubusercontent.com/11279744/145657073-5d70a59e-d63d-4b4c-af27-4bdb08e8c2c7.png">
<img width="1331" alt="Screen Shot 2021-12-10 at 4 30 33 PM" src="https://user-images.githubusercontent.com/11279744/145657074-a56b16d6-7037-49cd-bae0-be428f21936d.png">
<img width="1912" alt="Screen Shot 2021-12-10 at 4 31 05 PM" src="https://user-images.githubusercontent.com/11279744/145657075-d76bfa42-7eaf-4f7d-ba72-023f1de48548.png">
<img width="1912" alt="Screen Shot 2021-12-10 at 4 31 15 PM" src="https://user-images.githubusercontent.com/11279744/145657076-538cddc2-0c5a-49ef-b863-1bdbf1a34e10.png">



## QA steps

As user, check various edit forms and admin screens and verify new colors are not distracting, things still function as expected.

### Definition of Done


### Select Team for PR review


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?

Change management for editors will come closer to the turn-on date for this theme. For now we continue on unburdened and unbothered.
